homeassistant/components/shelly/services.yaml

```diff
@@ -0,0 +1,10 @@
+# shelly service descriptions.
+
+ota_update:
```

Comment:
I think we should add a flag named `beta` which will allow updating to beta firmware, since the OTA command support triggering update to beta FW.
https://shelly-api-docs.shelly.cloud/#ota
Might also need to consider if we allow specifying a URL for a direct update file in the service call

Reply:
I have added the `url` and `beta` parameter
In addition there is now a `force` parameter which allows to downgrade from beta-version to release-version and also to reinstall current version